views/doc.html

```diff
@@ -16,7 +16,7 @@
 
         <h1 class="post-title">{{ doc.title }}</h1>
 
-        <div class="post-content">
+        <div class="post-content has-components-table">
```

Comment:
No need to add this class, if you put it in the docs-specific SCSS file, like suggested.